apps/edxec/templates/services/app/secret.yml.j2

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: edxec
+    service: app
+  name: "{{ edxec_secret_name }}"
+  namespace: "{{ project_name }}"
+data:
+  local.yml: "{{ EDXEC_VAULT.LOCAL_FILE | default('') | to_nice_yaml | b64encode }}"
```

Comment:
Is it allow to have an empty local file ?

Reply:
Yes.